libraries/pbrlib/pbrlib_defs.mtlx

```diff
@@ -246,8 +246,46 @@
     <input name="exposure" type="float" value="0.0" doc="Exposure control for the light intensity" />
     <output name="out" type="lightshader" />
   </nodedef>
-
+   
   <!--
+    Node: unlit_surface
+    Uniform emission surface shader. Variants exist for the following input types: color3, color4, vec2, vec3, vec4, and float.
+  -->
+  <nodedef name="ND_unlit_surface_color4" node="unlit_surface" version="1.0" isdefaultversion="true" nodegroup="pbr" doc="Uniform emission surface shader which accepts a color4 input" >
+      <input name="color" type="color4" value="1, 1, 1, 1" />
```

Comment:
Since the new "unlit_surface" node supports scalar and vector types in addition to colors, would it make more sense for its first input to be named "value" instead of "color"?  This would mirror the naming convention for the first input of the "constant" node, which also supports scalar and vector types.

Reply:
This one is interesting as the second input is "opacity" so thought that "color" makes sense as even though the type may be vec3 it's interpreted as a color. 

"value" could work as well as I have no strong leaning either way but it would be "value" + "opacity"